services/dy-2Dgraph/TestingDBConnectionwithSimcoreSDK.ipynb

```diff
@@ -93,6 +107,175 @@
+    "from s3wrapper.s3_client import S3Client\n",
+    "\n",
+    "\n",
+    "class DbSettings(object):\n",
+    "    def __init__(self):\n",
+    "        self._db_config = db_config()\n",
+    "        self.db = create_engine(self._db_config.endpoint, client_encoding='utf8')\n",
+    "        self.Session = sessionmaker(self.db)\n",
+    "        self.session = self.Session()\n",
+    "\n",
+    "class S3Settings(object):\n",
+    "    def __init__(self):\n",
+    "        self._config = s3_config()\n",
+    "        self.client = S3Client(endpoint=self._config.endpoint,\n",
+    "            access_key=self._config.access_key, secret_key=self._config.secret_key)\n",
+    "        self.bucket = self._config.bucket_name\n",
+    "        self.client.create_bucket(self.bucket)\n",
+    "@tenacity.retry(wait=tenacity.wait_fixed(2), stop=tenacity.stop_after_attempt(5) | tenacity.stop_after_delay(20))\n",
+    "def init_db():\n",
+    "    db = DbSettings()    \n",
+    "    Base.metadata.create_all(db.db)\n",
+    "    return db\n",
+    "\n",
+    "@tenacity.retry(wait=tenacity.wait_fixed(2), stop=tenacity.stop_after_attempt(5) | tenacity.stop_after_delay(20))\n",
+    "def init_s3():\n",
+    "    s3 = S3Settings()\n",
+    "    return s3\n",
+    "db = init_db()\n",
+    "new_Pipeline = ComputationalPipeline()\n",
+    "db.session.add(new_Pipeline)\n",
+    "\n",
+    "db.session.commit()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "node_uuid = os.environ.get(\"SIMCORE_NODE_UUID\")\n",
+    "configuration={\n",
+    "    \"version\":\"0.1\",\n",
+    "    \"inputs\": [\n",
+    "        {\n",
+    "            \"key\": \"in_1\",\n",
+    "            \"label\": \"vm 1Hz\",\n",
+    "            \"desc\": \"these are computed data out of a pipeline\",\n",
+    "            \"type\": \"fileUrl\",\n",
+    "            \"value\": \"link.\" + node_uuid + \".in_1\",\n",
+    "            \"timestamp\": \"2018-05-23T15:34:53.511Z\"\n",
+    "        },\n",
+    "        {\n",
+    "            \"key\": \"in_2\",\n",
+    "            \"label\": \"all results 1Hz\",\n",
+    "            \"desc\": \"computed data out of a pipeline\",\n",
+    "            \"type\": \"fileUrl\",\n",
+    "            \"value\": \"link.\" + node_uuid + \".in_2\",\n",
+    "            \"timestamp\": \"2018-05-23T15:34:53.511Z\"\n",
+    "        }\n",
+    "    ],\n",
+    "    \"outputs\": [        \n",
+    "    ]\n",
+    "}\n",
+    "\n",
+    "#configuration = json.loads(json_configuration)\n",
+    "new_Node = ComputationalTask(pipeline_id=new_Pipeline.pipeline_id, node_id=node_uuid, input=configuration[\"inputs\"], output=configuration[\"outputs\"])\n",
+    "db.session.add(new_Node)\n",
+    "db.session.commit()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "metadata": {},
+   "outputs": [
```

Comment:
Try avoid committing outputs in notebooks ... Perhaps we can create a git hook that removes them automatically